doc/whats_new/v0.20.rst

```diff
@@ -93,11 +93,16 @@ cannot assure that this list is complete.)
 Known Major Bugs
 ----------------
 
-* :issue:`11924`: :class:`LogisticRegressionCV` with `solver='lbfgs'` and
-  `multi_class='multinomial'` may be non-deterministic or otherwise broken on
-  macOS. This appears to be the case on Travis CI servers, but has not been
-  confirmed on personal MacBooks! This issue has been present in previous
-  releases.
+* :issue:`11924`: :class:`linear_model.LogisticRegressionCV` with
+  `solver='lbfgs'` and `multi_class='multinomial'` may be non-deterministic or
+  otherwise broken on macOS. This appears to be the case on Travis CI servers,
+  but has not been confirmed on personal MacBooks! This issue has been present
+  in previous releases.
+
+* :issue:`9354`: :func:`metrics.pairwise.euclidean_distances` (which is used
+  several times throughout the library) gives results with poor precision,
+  which particularly affects its use with 32-bit float inputs. 32-bit floats
+  were handled with higher precision prior to version 0.19.
```

Comment:
Weren't they handled at all prior to this? Maybe say explicitly that it used to be cast to 64 bit?